### PR TITLE
Sjosey fix v1 java example

### PR DESCRIPTION
**Pull request by StephenJosey**
Updated libraries, and fixed the ItemManager. Also made the Payment Reporter use the previous year instead of a hard coded year. Let me know if anything should be changed/added.

### CODE REVIEW

File: connect-examples/v1/java/src/main/java/com/squareup/connectexamples/ItemManager.java [hunk 1]
```diff
@@ -95,18 +98,21 @@ public JSONObject updateItem(String itemId) {
       return null;
     }
 
+    if (response.getStatus() != 200) {
```
Comment (shaofu88) [inline connect-examples/v1/java/src/main/java/com/squareup/connectexamples/ItemManager.java]: Do you want to group your if statement with the following? Response could be n=ull like the following condition right? `response != null && response.getStatus() == 200`

File: connect-examples/v1/java/src/main/java/com/squareup/connectexamples/ItemManager.java [hunk 2]
```diff
@@ -115,6 +121,10 @@ public JSONObject deleteItem(String itemId) {
...
```
Comment (shaofu88) [inline connect-examples/v1/java/src/main/java/com/squareup/connectexamples/ItemManager.java]: same here

File: connect-examples/v1/java/src/main/java/com/squareup/connectexamples/PaymentsReporter.java [hunk 3]
```diff
@@ -206,10 +217,14 @@ public void printSalesReport(List<JSONObject> payments) {
 
   // Call the methods defined above
   public static void main(String[] args) {
-    PaymentsReporter reporter = new PaymentsReporter();
-    List<JSONObject> payments = reporter.get2015Payments(reporter.getLocationIds());
-    if (payments != null) {
-      reporter.printSalesReport(payments);
+    try {
```
Comment (shaofu88) [inline connect-examples/v1/java/src/main/java/com/squareup/connectexamples/PaymentsReporter.java]: why are you doing try & catch here? What error did you see locally?

Comment (StephenJosey) [reply connect-examples/v1/java/src/main/java/com/squareup/connectexamples/PaymentsReporter.java]: get2015Payments (which I will definitely rename now lol...) can throw an exception, so it must be in try block. The most common error I'm catching here is if they just forget to replace the credentials. Let me know if that doesn't make sense or if I'm doing it incorrectly, though!

File: connect-examples/v1/java/src/main/java/com/squareup/connectexamples/ItemManager.java (hunk 1 above)
Comment (shaofu88) [reply connect-examples/v1/java/src/main/java/com/squareup/connectexamples/ItemManager.java]: I mean do you want to add `response != null` to your if statement.

Comment (StephenJosey) [reply connect-examples/v1/java/src/main/java/com/squareup/connectexamples/ItemManager.java]: Good point, I'll add that shortly 👍

File: connect-examples/v1/java/src/main/java/com/squareup/connectexamples/PaymentsReporter.java (hunk 3 above)
Comment (shaofu88) [reply connect-examples/v1/java/src/main/java/com/squareup/connectexamples/PaymentsReporter.java]: okay that makes sense!

**Pull request merged**